tests/test_sanity.py

```diff
@@ -286,6 +286,29 @@ def test_llvm_fastcomp(self):
 
     self.check_working([EMCC, 'tests/hello_world.cpp', '-s', 'ASM_JS=0'], '''Compiler settings are incompatible with fastcomp. You can fall back to the older compiler core, although that is not recommended''')
 
+  def test_check_engine(self):
```

Comment:
this is a better test than I had in mind actually. it doesn't test `~/.emscripten` usage itself, but I guess your bots will ;) and it tests the rest in a cleaner way.

but given that, it's actually an ok test to put in `tests/test_other.py`, the sanity checks are literally for modifying the configuration. so please just move it to there.